thingsearch/model/src/main/java/org/eclipse/ditto/thingsearch/model/signals/commands/query/AggregateThingsMetrics.java

```diff
@@ -169,6 +172,7 @@ public Map<String, String> getGroupingBy() {
         return groupingBy;
     }
 
+    @Nullable
```

Comment:
Please also make that return an `Optional<String>` instead of using `@Nullable` - we don't do that for return types in the Ditto codebase - just for parameters.